pyecharts/echarts/label.py

```diff
@@ -0,0 +1,52 @@
+from pyecharts.echarts.json_serializable import JsonSerializable
+
+
+class Label(JsonSerializable):
+
+    def __init__(
+        self,
+        visibility,
+        position,
+        text_color,
+        text_size,
+        formatter=None,
+        chart_type=None
+    ):
+        self.config = {
+            "show": visibility,
+            "position": position,
+            "textStyle": {"color": text_color, "fontSize": text_size},
+        }
+
+        if chart_type != 'graph':
+            if formatter is None:
+                if type == "pie":
+                    formatter = "{b}: {d}%"
+        if formatter:
+            self.config['formatter'] = formatter
+
+
+EmphasisLabel = Label
```

Comment:
`EmphasisLabel` is a state of Label, relative to `NormalLabel`, not a alias.So class inheritance is a better choice.
```python
class EmphasisLabel(Label):
    pass
```